packages/manager/modules/pci/src/projects/project/training/jobs/jobs.scss

```diff
@@ -0,0 +1,5 @@
+.oui-button_link {
+  a {
+    text-decoration: none;
```

Comment:
I'm not sure this is accessible, the only visual way to know it's a link will be with the color/font-weight difference

Reply:
Without, we have a underline on the button and it's quite ugly. We know it's a link because we have the hover.

Reply:
As discussed together, I think we shouldn't have to override this style, the behaviour should be consistent across all manager if possible and we should try to avoid custom styles when we can. 
If you think that the display is ugly, may I suggest opening an issue or a discussion on https://github.com/ovh/ovh-ui-kit so we can see how the components can be improved

Reply:
👍